SimTracker/TrackAssociation/plugins/ResolutionModel.h

```diff
@@ -0,0 +1,33 @@
+#ifndef __RecoFTL_FastTimingKludge_ResolutionModel_h__
+#define __RecoFTL_FastTimingKludge_ResolutionModel_h__
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h